### PR TITLE
Update setup_env for new UI

### PR DESCRIPTION
**Pull request by jerith**
This replaces #469 which is a PR onto the now-obsolete develop-frontend.

### CODE REVIEW

File: go/base/management/commands/go_setup_env.py [hunk 1]
```diff
@@ -302,36 +256,55 @@ def assign_application(self, account, application_module):
...
-                    'foo': 'bar',
-                }
-            },
-            {
-                ...
-            }]
+    def setup_channels(self, user, channels):
+        user_api = vumi_api_for_user(user)
+        for channel in channels:
+            tag = tuple(channel.split(':'))
```
Comment (smn) [inline go/base/management/commands/go_setup_env.py:262]: do we want to expand this to make sure we only get two?

File: setup_env/workers.yaml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      riak_manager: *RIAK-MANAGER
+      send_to: *SEND-TO
+
+  jsbox:
+    class: go.apps.jsbox.vumi_app.JsBoxApplication
+    config:
+      metrics_prefix: "go.accounts."
+      env:
+        NODE_PATH: "/usr/lib/node_modules"
+      app_context: "{require: function(m) { if (m == 'jed' || m == 'vumigo_v01') return require(m); return null; }, Buffer: Buffer}"
```
Comment (smn) [inline setup_env/workers.yaml:161]: `q` & `libxmljs` are missing from this list.

**Comment by smn** [comment]
:+1: other than the two minor comments

File: go/base/management/commands/go_setup_env.py (hunk 1 above)
Comment (jerith) [inline go/base/management/commands/go_setup_env.py:262]: We could, but the error will be caught later anyway so it isn't that big a deal.

File: setup_env/workers.yaml (hunk 2 above)
Comment (jerith) [inline setup_env/workers.yaml:161]: This is copied directly from what was there at the time. I'll check what's there now and update accordingly.

Comment (jerith) [inline setup_env/workers.yaml:161]: Yes, this is what we have in develop. I'm inclined to leave it as-is and fix it later.

**Pull request merged**